docs/docs/dev/networking/cni.md

```diff
@@ -1,44 +1,270 @@
+  --set "cni.confName=10-flannel.conflist"
 ```
 
 :::
 ::: tab "Helm"
 
-When using [Helm](../../installation/helm), we should use the values in the `cni` section to set the relevant parameters.
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/var/lib/rancher/k3s/agent/etc/cni/net.d,cni.binDir=/bin,cni.confName=10-flannel.conflist \
+   kuma kuma/kuma
+```
+
+:::
+::::
+:::::
+
+::::: tab "Kind"
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/opt/cni/bin" \
+  --set "cni.confName=10-kindnet.conflist"
+```
+
+:::
+::: tab "Helm"
 
 ```shell
 helm install --namespace kuma-system \
-  --set cni.enabled=true,cni.chained=true,cni.netDir="/etc/cni/net.d",cni.binDir=/opt/cni/bin,cni.confName=10-calico.conflist \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-kindnet.conflist \
+   kuma kuma/kuma
+```
+
+:::
+::::
+:::::
+
+::::: tab "Azure"
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/opt/cni/bin" \
+  --set "cni.confName=10-azure.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-azure.conflist \
+   kuma kuma/kuma
+```
+
+:::
+::::
+:::::
+
+::::: tab "AWS - EKS"
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/opt/cni/bin" \
+  --set "cni.confName=10-aws.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-aws.conflist \
    kuma kuma/kuma
 ```
 
 :::
 ::::
+:::::
+
+::::: tab "Google - GKE"
+
+You need to [enable network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) in your cluster (for existing clusters this redeploys the nodes).
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/home/kubernetes/bin" \
+  --set "cni.confName=10-calico.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/home/kubernetes/bin,cni.confName=10-calico.conflist \
+   kuma kuma/kuma
```

Comment:
To be consistent with other instructions where we are using multiple `--set` flages, I would go with:
```suggestion
helm install \
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/etc/cni/net.d" \
  --set "cni.binDir=/home/kubernetes/bin" \
  --set "cni.confName=10-calico.conflist" \
   kuma kuma/kuma
```